docs/best_practices/extensions.rst

```diff
@@ -1,39 +1,49 @@
 Extensions
 ==========
 
-Extend only when necessary. Extensions are an essential mechanism to integrate data with NWB that is otherwise not
-supported. However, we here need to consider that there are certain costs associated with extensions, e.g., cost of
-creating, supporting, documenting, and maintaining new extensions and effort for users to use and learn extensions.
-As such, users should attempt to use core neurodata_types or existing extentions before creating extensions.
-``DynamicTables``, which are used throughout the NWB schema e.g., to store information about time intervals and
-electrodes, provide the ability to dynamically add columns without the need for extensions, and can help avoid the
-need for custom extensions in many cases.
+Extend existing NWBFile structures only when absolutely necessary. Extensions are an essential mechanism to integrate
```

Comment:
I don't like the use of "absolutely" here. It sounds like we are discouraging users from creating extensions